Drive/rover_drive_system.hpp

```diff
@@ -22,10 +23,12 @@ class RoverDriveSystem
   /// Mission controls possible input values
   struct MissionControlData
   {
-    bool is_operational;
+    int is_operational;
     char drive_mode;
     float rotation_angle;
     float speed;
+    char response_body[300];
+    std::string request_parameter;
```

Comment:
Ideally would like to have these both as strings but response_body uses sscanf. I guess I could just alter it but wanted to know what @kammce thought first?

Reply:
You can make them both character arrays if you like. I mentioned that you can use `snprintf` below, which is a better choice in terms of performance and eliminating memory fragmentation (does not dynamically allocate memory)